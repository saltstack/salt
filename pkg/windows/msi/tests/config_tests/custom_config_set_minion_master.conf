# User custom config line 1/3
#master: custom.master
# User custom config line 2/3
#id: custom.minion
# User custom config line 3/3
