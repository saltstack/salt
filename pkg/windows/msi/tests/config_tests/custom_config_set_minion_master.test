properties START_MINION="" CONFIG_TYPE=Custom CUSTOM_CONFIG=tests\config_tests\custom_config_set_minion_master.conf MASTER=cli.master MINION_ID=cli.minion
dormant
