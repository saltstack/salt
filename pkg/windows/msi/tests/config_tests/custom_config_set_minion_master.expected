# User custom config line 1/3
#master: custom.master
master: cli.master
# User custom config line 2/3
#id: custom.minion
id: cli.minion
# User custom config line 3/3
